### PR TITLE
Refine UI and Remove wazuh prefix

### PR DESCRIPTION
**Pull request by JuanGarriuz**
### Description

This PR involves removing unmaintained visualizations and improving the titles and references across the application.
 
### Issues Resolved

https://github.com/wazuh/wazuh-dashboard-plugins/issues/6128

#### About 
image
#### Logs
image
#### Health Check
image
#### Deploy New Agent
image
#### Endpoint Summary chart remove
image
image
image

### Test
- [ ] Check that the Endpoints Summary does not display the evolution chart and ensure that the status and details adjust correctly within the space. 
- [ ] Verify that the Wazuh prefix has been organically and appropriately removed where necessary.

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by Desvelao** [comment]
**thought**: the usage of generic terms such as `server` or API is not clear to me. There are 3 servers in a stack (manager, dashboard and indexer) that have APIs.

File: plugins/main/public/components/health-check/services/check-api.service.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ const trySetDefault = async (checkLogger: CheckLogger) => {
         for (var j = 0; j < errors.length; j++) {
           if (errors[j].includes('ERROR3099 - 405: Method Not Allowed')) {
             return Promise.reject(
-              `No API available to connect. This may be related to a version mismatch between Wazuh Server and ${PLUGIN_APP_NAME}. Please check the versions and try again. Read more about this in our troubleshooting guide: ${webDocumentationLink(
+              `No API available to connect. This may be related to a version mismatch between Server and ${PLUGIN_APP_NAME}. Please check the versions and try again. Read more about this in our troubleshooting guide: ${webDocumentationLink(
```
[the remark this answers is not in the document]
Comment (JuanGarriuz) [reply plugins/main/public/components/health-check/services/check-api.service.ts:57]: In this case, it is being used as a proper name to refer to the Wazuh server.

[the remark this answers is not in the document]
Comment (Tostti) [reply plugins/main/public/components/health-check/services/check-api.service.ts:57]: I also think that this should be lowercase. `between the server and the Dashboard`

**Review by Tostti — COMMENTED** [review]
# Test
image
image
image
image
image
image

File: plugins/main/public/components/settings/about/generalInfo.tsx [hunk 2]
```diff
@@ -15,80 +15,94 @@ interface SettingsAboutGeneralInfoProps {
...
       </EuiPanel>
     </EuiFlexItem>
   );
 
   return (
-    <EuiFlexGroup gutterSize="l" direction="row" responsive>
+    <EuiFlexGroup gutterSize='l' direction='row' responsive>
       <Section
-        title={`Welcome to the ${pluginAppName}`}
+        title={`Welcome to the dashboard`}
```
Comment (Desvelao) [inline plugins/main/public/components/settings/about/generalInfo.tsx:46]: **nitpick**: this was using a variable, why did we change to hardcoded string?

File: plugins/main/common/api-info/endpoints.json
Comment (Desvelao) [inline plugins/main/common/api-info/endpoints.json:1]: **nitpick**: undo these changes. This files is generated through an script.

**Comment by Desvelao** [comment]
Reviewing the source code, I found some references that we could consider to change:
- `PLUGIN_PLATFORM_NAME` constant
- `WAZUH_INDEXER_NAME` constant
- `Wazuh not ready yet` reference related to server/agent configuration

files:
- plugins/main/public/components/add-modules-data/guides/audit.js
- plugins/main/public/components/add-modules-data/guides/fim.js
- plugins/main/public/controllers/management/components/management/configuration/github/github.tsx
- plugins/main/public/controllers/management/components/management/configuration/google-cloud-pub-sub/google-cloud-pub-sub.js
- plugins/main/public/controllers/management/components/management/configuration/office365/constants.tsx
- plugins/main/public/controllers/management/components/management/configuration/utils/wz-fetch.js
- plugins/main/public/controllers/settings/settings.js
- plugins/main/public/services/check-daemon-status.js
- plugins/main/server/controllers/wazuh-api.ts
- plugins/wazuh-core/common/constants.ts

**Comment by Desvelao** [comment]
# Test

image
image
image
image
image

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by asteriscos — APPROVED** [review]
